euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/MapElementsTest.java

```diff
@@ -39,7 +40,7 @@ public void testOnTwoPartitions() {
       @Override
       protected Dataset<String> getOutput(Dataset<Integer> input) {
         return MapElements.of(input)
-            .using(String::valueOf)
+            .using((UnaryFunction<Integer, String>) String::valueOf)
```

Comment:
This is related specifically to `String#valueOf` which is heavily overloaded and offers different number of arguments. It shouldn't cause any problem in a general usage.